src/Corrade/Utility/ConfigurationValue.cpp

```diff
@@ -49,7 +49,7 @@ namespace Implementation {
     }
 
     template<class T> T BasicConfigurationValue<T>::fromString(const std::string& stringValue, ConfigurationValueFlags flags) {
-        std::string _stringValue = stringValue;
+        const std::string& _stringValue = stringValue;
```

Comment:
Wow. Why did I ever do this? This speeds up parsing of all configuration files and command-line arguments by ∞ :D